packages/office-addin-mock/src/officeMockObject.ts

```diff
@@ -69,6 +71,25 @@ export class OfficeMockObject {
     this[propertyName] = this.properties.get(propertyName)?.value;
   }
 
+  /**
+   * Adds a property of any type to OfficeMockObject
+   * @param propertyName Property name to the property to be added
+   * @param value Value this added property will have
+   */
+  setProperty(propertyName: string, value: any) {
```

Comment:
Before we got and just change this we need to confirm with others that this is the right name and that it should work for both properties and functions.

We don't want to get into further trouble by changing the name so easily and frequently.

Reply:
Sure. Do you have some names in mind for us to confirm?

Reply:
I don't have specific names in mind, I just think the audience on approval of the new names and the timeline for releasing them needs to be more then just the two of us and while others are invited to look at these reviews, frequently they don't have the time :-).  You should get buy off in a broader audience email.